libraries/fc/include/fc/int_array.hpp

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include <fc/variant.hpp>
+
+#include <limits>
+#include <vector>
+
+namespace fc {
+
+/**
+ * Like fc::array, but does zero-initialization and has friendly JSON format
+ */
+
+template<typename T, size_t N>
+class int_array
+{
+    public:
+       int_array(){ memset( data, 0, sizeof(data) ); }
+
```

Comment:
We should allow instantiation of an `int_array< T, N >` from `array< T, N >`.

For example, the witness schedule in `witness_schedule_object` to printing in `api_witness_schedule_object`.